src/client/index.ts

```diff
@@ -20,7 +20,7 @@ export default abstract class Provider {
     throw new Error('Function configure has not been defined')
   }
 
-  getSchema(): string {
+  getSchema(): any {
```

Comment:
can we type this a little better anyway?

Reply:
done!